packages/cheerio-crawler/src/internals/cheerio-crawler.ts

```diff
@@ -935,47 +241,9 @@ function extractUrlsFromCheerio($: CheerioRoot, selector: string, baseUrl?: stri
         .filter((href) => !!href) as string[];
 }
 
-/**
- * The stream object returned from got does not have the below properties.
- * At the same time, you can't read data directly from the response stream,
- * because they won't get emitted unless you also read from the primary
- * got stream. To be able to work with only one stream, we move the expected props
- * from the response stream to the got stream.
- * @internal
- */
-function addResponsePropertiesToStream(stream: GotRequest) {
-    const properties = [
-        'statusCode', 'statusMessage', 'headers',
-        'complete', 'httpVersion', 'rawHeaders',
-        'rawTrailers', 'trailers', 'url',
-        'request',
-    ];
-
-    const response = stream.response!;
-
-    response.on('end', () => {
-        // @ts-expect-error
-        Object.assign(stream.rawTrailers, response.rawTrailers);
-        // @ts-expect-error
-        Object.assign(stream.trailers, response.trailers);
-
-        // @ts-expect-error
-        stream.complete = response.complete;
-    });
-
-    for (const prop of properties) {
-        if (!(prop in stream)) {
-            // @ts-expect-error
-            stream[prop] = response[prop as keyof IncomingMessage];
-        }
-    }
-
-    return stream as unknown as IncomingMessage;
-}
-
 /**
  * Creates new {@apilink Router} instance that works based on request labels.
- * This instance can then serve as a `requestHandler` of your {@apilink CheerioCrawler}.
+ * This instance can then serve as a `requestHandler` of your {@link CheerioCrawler}.
```

Comment:
use `@apilink` in all doc comments, we need that for API docs to respect versioning

Reply:
I must have forgotten about this one. 448e3a75aa91945e85427806b9d0429adc66683d